### ISSUE TITLE
RequesterUtils references Headers type but does not import/define it

### ISSUE THREAD

**Issue by thesuavehog** [jdalrymple/gitbeaker#2545]
**Description**

- Node.js version: v14.19.0
- Gitbeaker version: 35.6.0
- Gitbeaker release (cli, node, browser, core, requester-utils): requester-utils
- OS & version: Windows 11
- Typescript version: 4.6.2

When building my project with `@gitbeaker` the `RequesterUtils` does not compile in Typescript because it uses a `Headers` interface which is not imported or defined.

```
❯ tsc
node_modules/@gitbeaker/requester-utils/dist/types/RequesterUtils.d.ts:28:39 - error TS2304: Cannot find name 'Headers'.

28     headers: Record<string, string> | Headers;
                                         ~~~~~~~

node_modules/@gitbeaker/requester-utils/dist/types/RequesterUtils.d.ts:40:40 - error TS2304: Cannot find name 'Headers'.

40     headers: Record<string, unknown> | Headers;
                                          ~~~~~~~
```

**Steps to reproduce**

Not sure if this is specific to the `tsconfig` I have but I just installed the library and ran `tsc` to compile the project.

**Expected behaviour**

The compilation of a project using `@gitbeaker/requester-utils` should compile successfully.

**Actual behaviour**

Typescript compile fails.

**Possible fixes**

`import FormData, { Headers } from 'form-data';`

**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`35.6.1`](https://github.com/jdalrymple/gitbeaker/releases/tag/35.6.1) :rocket:

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`35.6.1`](https://github.com/jdalrymple/gitbeaker/releases/tag/35.6.1) :rocket:

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`35.6.1`](https://github.com/jdalrymple/gitbeaker/releases/tag/35.6.1) :rocket:
